react/Alerter/index.jsx

```diff
@@ -22,7 +97,11 @@ class Alerter extends Component {
 
       this.closeTimer = setTimeout(() => {
         this.setState({ hidden: true })
-        if (this.props.onClose && typeof this.props.onClose === 'function') this.props.onClose()
+        if (this.props.onClose && typeof this.props.onClose === 'function') {
```

Comment:
If I'm not mistaken, `props.onClose` is now always `dismiss`, so it's always set and always a function and this check isn't needed anymore?